about.html

```diff
@@ -0,0 +1,132 @@
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+        <div class="logo " id="">
+          <a href="index.html"><img src="./assets/images/navlogo.svg" alt=""></a>
+        </div>
+        <nav class="menu" id="menu">
+          <ul class="mobile-menu">
+            <li class="nav-item  primary-font "><a href="index.html" class="nav-link primary-font menuaction">Home</a></li>
+            <li class="nav-item  primary-font "><a href="#" class="nav-link menuaction">Program</a></li>
+            <li class="nav-item  primary-font "><a href="#about" class="nav-link menuaction">Join</a></li>
+            <li class="nav-item primary-font "><a href="#about" class="nav-link menuaction">Sponsor</a></li>
+            <li class="nav-item primary-font"><a href="#about" class="nav-link menuaction">News</a></li>
+          </ul>
+        </nav>
+        <a href="#campaign" class="campaign campaign-link tetiary-font">CC Campaign</a>
+      </nav>
+      
+     
+    </div>
+  
+    <div class="about-header-body">
+      <p class="about-headline-header primary-font">"Hello! Sharing world"</p>
+      <p class="about-headline-body-about secondary-font">CREATIVE WRITERS GLOBAL SUMMIT 2015</p>
+    </div>
+    <div class="headline-body-intro">
+      <p class="about-body-intro primary-font">The Global Summit brings together the community of writters in the  financial worlds, Intellectual, academics,  and activists who comprise the Creative Writers affiliate network in a different country every two years.
+        This year, we hope to expand our invitation list by including organizations and individuals who want to work with us on shared projects that advance the cause of the in every espect of life
+      </p>
+    </div>
+    
+    <div class="contact-submit secondary-font">
+      <p class="contact-body-intro primary-font">Please contact us per Email for nay further questions about CC Global Summit 2015!</p>
+      <a href="ccsummit2015@ccNigeria.org" class="about-connect primary-font">ccsummit2015@ccNigeria.org</a>
+      <P ></P>
+    </div>
+  </header>
+  <section class="logo-program-section">
+    <div class="logo-section">
+      <div class="logo-creation">
+       <p class="logo-creation-title tetiary-font">Creative Writers 
+        Global Summit 2015 Logo</p> 
+        <p class="logo-creation-body primary-font">The logo of CC Global Summit 2015 was decided through the logo competition from 8, June to 7, July.
+        </p>
+      </div>
+      
+    </div>
+    <img src="./assets/images/logo.svg" alt="" class="logoImg">
+  </section>
+  <section class="past-program">
+    <div class="past-section">
+      <h1 class="past-program-header tetiary-font">See the past CC Global Summits</h1>
+      <p class="past-program-body primary-font">Take a look at the past two CC Global Summits which took place in Buenos Aires and in Warsaw</p>
+    </div>
```

Comment:
- [ Optional] Consider using the fonts(size, weight, color) as provided from the design because the texts beside the speakers are looking very small on your desktop size and its hard to read them. Implement that to make your website more responsive 
![image](https://user-images.githubusercontent.com/43877689/227730350-ad45b2c6-1653-4117-88a2-a1a02b77d707.png)

Reply:
hello @ishimwezachee are we ask to show the code or to show the design in the site developed? As i could not find this in the instruction given.

Thanks

Reply:
Let's have a chat on slack if you ok with that